index.html

```diff
@@ -207,23 +259,30 @@ <h3>
         &lt;script&gt;
           const button = document.querySelector("button");
           button.addEventListener("click", async () =&gt; {
+            const protocol = "example-request-protocol";
+            // Check for DC API and protocol support
+            if (!DigitalCredential.userAgentAllowsProtocol(protocol)) {
+              // Protocol not allowed. Fall back to a different verification method.
```

Comment:
```suggestion
              // The browser will not permit the use of this protocol.
              // Fall back to a different verification method.
```

Reply:
I want to avoid using “permit” as it means something different to “allow”, and also to keep it consistent with the method’s name.

Reply:
Sorry for the confusion. I used your suggestion to merge it in.